pkg/sql/catalog/bootstrap/initial_values.go

```diff
@@ -132,56 +132,30 @@ func (f hardCodedInitialValues) build(
 	return kvs, splits, nil
 }
 
-var (
-	// system231 is the hard-coded string representation of the
-	// initial values for a system tenant as produced by the 23.1
-	// release binary.
-	//
-	// This file was generated by copying over the expected results for the
-	// TestInitialValuesToString test in the release-23.1 branch.
-	//go:embed data/23_1_system.keys
-	system231 string
-
-	// system231SHA256 is the SHA-256 hash of the above constant.
-	//go:embed data/23_1_system.sha256
-	system231SHA256 string
-
-	// nonSystem231 is the hard-coded string representation of the
-	// initial values for a non-system tenant as produced by the 23.1
-	// release binary.
-	//
-	// This file was generated by copying over the expected results for the
-	// TestInitialValuesToString test in the release-23.1 branch.
-	//go:embed data/23_1_tenant.keys
-	nonSystem231 string
-
-	// nonSystem231SHA256 is the SHA-256 hash of the above constant.
-	//go:embed data/23_1_tenant.sha256
-	nonSystem231SHA256 string
-
-	// system232 is the hard-coded string representation of the
-	// initial values for a system tenant as produced by the 23.2
-	// release binary.
-	//
-	// This file was generated by copying over the expected results for the
-	// TestInitialValuesToString test in the release-23.2 branch.
-	//go:embed data/23_2_system.keys
-	system232 string
-
-	// system232SHA256 is the SHA-256 hash of the above constant.
-	//go:embed data/23_2_system.sha256
-	system232SHA256 string
-
-	// nonSystem232 is the hard-coded string representation of the
-	// initial values for a non-system tenant as produced by the 23.2
-	// release binary.
-	//
-	// This file was generated by copying over the expected results for the
-	// TestInitialValuesToString test in the release-23.2 branch.
-	//go:embed data/23_2_tenant.keys
-	nonSystem232 string
-
-	// nonSystem232SHA256 is the SHA-256 hash of the above constant.
-	//go:embed data/23_2_tenant.sha256
-	nonSystem232SHA256 string
-)
+// The following variables hold hardcoded bootstrap data for older versions (as
+// produced by the final release version). For each version, we have system and
+// non-system keys, and each set of keys has an associated SHA-256.
```

Comment:
it was kind of nice that the old comment said you how you should populate new values when the version changes. could there be instructions on how to do that with the new way?

Reply:
@andyyang890 is adding a comment in #114120

I will also rewrite the runbooks for bumping versions and check them in as an `.md` file, and I will point to that here.